gcodeutils/gcode_optimize_arcs.py

```diff
@@ -44,7 +44,7 @@ def main():
     GCodeArcOptimizerFilter().filter(gcode)
 
     # write back modified gcode
-    gcode.write(args.outfile if args.inplace is True else args.outfile)
+    gcode.write(open(args.infile.name, 'w') if args.inplace is True else args.outfile)
```

Comment:
isn't `gcode.write(args.infile.name if args.inplace else args.outfile)` enough to have inplace editing working ?